rendering/fbx.ksy

```diff
@@ -0,0 +1,148 @@
+meta:
+  id: fbx
+  title: FBX 3D model interchange format
```

Comment:
The word `format` is usualy redundant. Maybe just `Kaydara FiLMBOX 3D Model`?

Reply:
I think I'll keep normal capitalisation of Filmbox but otherwise agreed.